shortcircuit-api/src/main/java/com/powsybl/shortcircuit/json/ShortCircuitParametersSerializer.java

```diff
@@ -48,6 +48,7 @@ public void serialize(ShortCircuitParameters parameters, JsonGenerator jsonGener
             serializerProvider.defaultSerializeField("voltageRanges", parameters.getVoltageRanges(), jsonGenerator);
         }
         jsonGenerator.writeBooleanField("detailedReport", parameters.isDetailedReport());
+        jsonGenerator.writeStringField("debugDir", parameters.getDebugDir());
```

Comment:
Can write this field optionally (not serialize a null value)?

Reply:
DONE in https://github.com/powsybl/powsybl-core/pull/3455/commits/7c47658f6b28d0361b4d65f9616aa95e05ceb679